MVP Module/Tests/Feed UI/FeedUIIntegrationTests.swift

```
@@ -47,6 +47,32 @@ final class FeedUIIntegrationTests: XCTestCase {
 		XCTAssertFalse(sut.isShowingLoadingIndicator, "Expected no loading indicator once user initiated loading completes with error")
 	}
 	
+	func test_loadFeed_BadRequestError() {
```

Missing test/assertions to check that:

1) When the user reloads after an error (pull to refresh), the error is hidden.
2) When the user taps on the button, the error is hidden.